R/Features-constructors.R

```diff
@@ -25,33 +37,34 @@
 ##' @param ... Further arguments that can be passed on to `read.csv`
 ##'     except `stringsAsFactors`, which is always `FALSE`.
 ##'
-##' @param name An `character(1)` to name assay. If not set,
-##'     `features` is used.
+##' @param name An `character(1)` to name assay in the `Features` 
+##'     object. If not set, `features` is used.
 ##'
-##' @return An instance of class [Features].
+##' @return An instance of class [Features] or [SummarizedExperiment].
 ##'
 ##' @author Laurent Gatto
 ##'
 ##' @importFrom utils read.csv
 ##' @import SummarizedExperiment
 ##'
-##' @seealso The [Features] class for an example on how to manipulate use
+##' @seealso The [Features] class for an example on how to use
 ##'     `readFeatures` and how to further manipulate the resulting data.
 ##'
 ##' @importFrom methods new validObject
 ##'
 ##' @md
+##' @aliases readFeatures
```

Comment:
If the man page is about `readFeatures`, then you need to add `readSummarizedExperiment` as an alias. Otherwise it's ok.